yaml/_type/pairs.ts

```diff
@@ -25,19 +25,16 @@ function resolveYamlPairs(data: unknown[][]): boolean {
 
   return true;
 }
-function constructYamlPairs(data: string) {
+function constructYamlPairs(
+  data: Record<string, unknown>[] | null,
+): [string, unknown][] {
   if (data === null) return [];
```

Comment:
```suggestion
  return data !== null ? data.flatMap(Object.entries) : [];
```

Reply:
Cool reduction, though it is not immediately obvious what is going on. I think I'll add an example doc.